client/components/site-selector/use-onboarding-url.ts

```diff
@@ -0,0 +1,35 @@
+import config from '@automattic/calypso-config';
+import { useLocale } from '@automattic/i18n-utils';
+import { useExperiment } from 'calypso/lib/explat';
+import isJetpackCloud from 'calypso/lib/jetpack/is-jetpack-cloud';
+
+const GUTENBOARDING_LOCALES = [ 'en', 'en-gb' ];
```

Comment:
Maybe I'm missing some context here, but can we reuse this instead?

https://github.com/Automattic/wp-calypso/blob/c7e24c921d38445d6814f165d1bd8ca26cfe75fa/packages/i18n-utils/src/locales.ts#L25

Reply:
Technically there's no reason why the Gutenboarding locales have to be English. If we found during testing that French performed just as well as English then it could be `[ 'en', 'en-gb', 'fr' ]`. Which is a reason to use our own special array: `englishLocales` and `GUTENBOARDING_LOCALES` are semantically different.

But for all intents and purposes I think the decision about which locales go to Gutenboarding _was_ made on their "english-ness". So yeah, I like the idea of using the pre-defined `englishLocales` variable.

Reply:
I think there's an argument for either, I implemented it this way because I coppied what's in https://github.com/Automattic/wp-calypso/blob/trunk/client/state/selectors/get-onboarding-url.js